test/core/Baked.Test.Architecture/Architecture/Feature/UsingPhaseArtifacts.cs

```diff
@@ -12,7 +12,7 @@ public void Configure(LayerConfigurator configurator)
         {
             configurator.Configure((string target) =>
             {
-                Artifact = configurator.Context.Get<int>();
+                configurator.Use<int>(@int => Artifact = @int);
```

Comment:
check documentation, this might occur in documentation and need to be updated as well...